spec/models/benefit_group_spec.rb

```diff
@@ -613,6 +613,41 @@
   end
 
   describe BenefitGroup, dbclean: :after_each do
+    let(:benefit_group) { FactoryGirl.create(:benefit_group)}
+    let(:carrier_profile) { FactoryGirl.create(:carrier_profile) }
```

Comment:
if we look through how group size factors are applied we see that the carrier_profile would require plan_option_kind 'sole_source'